packages/services/src/services/VideoPlayer/VideoPlayer.js

```diff
@@ -1,3 +1,6 @@
+import { AnalyticsAPI } from '@carbon/ibmdotcom-services';
```

Comment:
This can be pulled relatively since it's in the same package

Reply:
oh whoops yep done